l10n_ch_scan_bvr/wizard/scan_bvr.py

```diff
@@ -0,0 +1,431 @@
+        if partner.supplier_invoice_default_product:
+            prod = partner.supplier_invoice_default_product
+            product_onchange_result = invoice_line_model.product_id_change(
+                prod.id,
+                uom_id=False,
+                qty=0,
+                name='',
+                type='in_invoice',
+                partner_id=partner.id,
+                fposition_id=False,
+                price_unit=False,
+                currency_id=False,
+                company_id=None
+            )
+            # We will check that the tax specified
+            # on the product is price include or amount is 0
+            if product_onchange_result['value']['invoice_line_tax_id']:
+                taxes = self.env['account.tax'].browse(
+                    product_onchange_result['value']['invoice_line_tax_id'])
+                for taxe in taxes:
+                    if not taxe.price_include and taxe.amount != 0.0:
+                        raise UserError(
+                            _('The default product in this partner has '
+                              'wrong taxes configuration')
+                        )
+            account = product_onchange_result['value']['account_id']
+            taxes = product_onchange_result['value']['invoice_line_tax_id']
+            invoice_line_vals = {
+                'product_id': prod.id,
+                'account_id': account,
+                'name': product_onchange_result['value']['name'],
+                'uos_id': product_onchange_result['value']['uos_id'],
+                'price_unit': data['bvr_struct']['amount'],
+                'invoice_id': data['invoice_id'],
+                'invoice_line_tax_id': [(6, 0, taxes)],
+            }
+            invoice_line = invoice_line_model.create(invoice_line_vals)
+        return invoice_line
+
+    @api.multi
+    def _create_direct_invoice(self, data):
+        # We will call the function, that create invoice line
+        invoice_model = self.env['account.invoice']
+        invoice_tax_model = self.env['account.invoice.tax']
+        currency_model = self.env['res.currency']
+        partner_bank_model = self.env['res.partner.bank']
+        if data['bank_account']:
+            account_info = self.env['res.partner.bank'].browse(
+                data['bank_account'])
+        # We will now search the currency_id
+        currency = currency_model.search(
+            [('name', '=', data['bvr_struct']['currency'])])
+        # Account Modification
+        partner_bank = partner_bank_model.browse(data['bank_account'])
+        if data['bvr_struct']['domain'] == 'name':
+            partner_bank.write(
+                {'ccp': data['bvr_struct']['beneficiaire']})
+        else:
+            partner_bank.write(
+                {'bvr_adherent_num': data['bvr_struct']['bvrnumber'],
+                 'ccp': data['bvr_struct']['beneficiaire']})
+        date_due = time.strftime('%Y-%m-%d')
```

Comment:
Please use `fields.Date` helper `today` to use the right string format from constants

https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.fields.Date